app/controllers/api/admin/articles_controller.rb

```diff
@@ -5,8 +5,19 @@ class Api::Admin::ArticlesController < ApplicationController
   before_action :editor?
 
   def index
-      article = Article.where(published: false)
-      render json: article, each_serializer: Admin::Article::IndexSerializer
+    article = Article.where(published: false)
+    render json: article, each_serializer: Admin::Article::IndexSerializer
+  end
+
+  def show
+    article = Article.find(params[:id])
+    if article.published == true
+      render json: { message: 'This article was already published' }, status: 400
```

Comment:
Http 400 response is a bit weird for this, think 422 is better....

Reply:
Hm... both works but I think 400, 'bad request' is more suiting. You're not supposed to ask for published articles here.

Reply:
Well, okay, now that I think of it, I think you're right. Bad request is more about bad syntax I guess, and unprocessable entity can very well be good syntax but not something that we will process.